kubernetes_state/test_kubernetes_state.py

```diff
@@ -102,6 +122,38 @@ def test__update_kube_state_metrics(self, mock_poll):
 
         self.assert_resourcequota()
 
+    @mock.patch('checks.prometheus_check.PrometheusCheck.poll')
+    def test__update_kube_state_metrics_v040(self, mock_poll):
+        f_name = os.path.join(os.path.dirname(__file__), 'ci', 'fixtures', 'prometheus', 'protobuf.bin')
+        with open(f_name, 'rb') as f:
+            mock_poll.return_value = ('application/vnd.google.protobuf', f.read())
+
+        config = {
+            'instances': [{
+                'host': 'foo',
+                'kube_state_url': 'http://foo',
+            }]
+        }
+
+        self.run_check(config)
+
+        self.assertServiceCheck(NAMESPACE + '.node.ready', self.check.OK)
+        self.assertServiceCheck(NAMESPACE + '.node.out_of_disk', self.check.OK)
+        self.assertServiceCheck(NAMESPACE + '.pod.phase.running', self.check.OK)
+        self.assertServiceCheck(NAMESPACE + '.pod.phase.pending', self.check.WARNING)
+        # TODO: uncomment when any of these are in the test protobuf.bin
+        #self.assertServiceCheck(NAMESPACE + '.pod.phase.succeeded', self.check.OK)
```

Comment:
If `pod.phase.succeeded`, `pod.phase.failed` and `pod.phase.unknown` are tested in `test__update_kube_state_metrics` - Could they be removed from the comment ? Or could the comment be updated to mention protobuf1.x.x.bin ?

Reply:
Will look at that.